test/unit/dropbox-suite.js

```diff
@@ -290,6 +296,30 @@ define(['bluebird', 'requirejs', 'test/behavior/backend', 'test/helpers/mocks'],
       },
 
       {
+        desc: "#configure fetches the user info when no userAddress is given",
+        run: function (env, test) {
+          env.rs.widget = { view: { setUserAddress: function() {} } };
+
+          env.client.on('connected', function() {
+            test.assert(env.client.userAddress, 'John Doe');
```

Comment:
This is not really the user address. Can we get both the name and email address for the account maybe?